internal/cli/config/config.go

```diff
@@ -59,10 +59,10 @@ type Dac struct {
 }
 
 type Config struct {
-	RestClientConfig perseshttp.RestConfigClient `json:"rest_client_config"`
-	Project          string                      `json:"project,omitempty"`
-	RefreshToken     string                      `json:"refresh_token,omitempty"`
-	Dac              Dac                         `json:"dac,omitempty"`
+	RestClientConfig perseshttp.RestConfigClient `json:"rest_client_config,omitempty" yaml:"rest_client_config,omitempty"`
```

Comment:
Are you sure that everything is well hidden if a secret from there. Do we have some ?

Reply:
Ah yeah you are right. I need to work more on this part and use more what we did in the API to hidde secret.